lianad/src/lib.rs

```diff
@@ -76,7 +76,14 @@ pub struct Version {
 
 impl fmt::Display for Version {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{}.{}.{}-dev", self.major, self.minor, self.patch)
+        write!(
+            f,
+            "{}.{}.{}-dev-{}",
```

Comment:
```suggestion
            "{}.{}.{}-{}",
```
I'd drop `-dev` wdyt @edouardparis